Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -3351,7 +3349,8 @@ - (void)_singleTapRecognized:(UITapGestureRecognizer *)gestureRecognizer
         [self becomeFirstResponder];
     }
 
-    ASSERT(_potentialTapInProgress);
+    if (!_potentialTapInProgress)
```

Comment:
I wonder if we should move this early return up, to before we try and make the content view first responder. (Not really a new issue, but it might be good to fix while we're generalizing this mechanism beyond Live Text)